aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -243,7 +243,19 @@ class DispatchTable final {
               "Available functions are ", listAllDispatchKeys())
       }
 
-      const std::string dispatch_key_str = dispatch_key.has_value() ? toString(*dispatch_key) : "None";
+      if (dispatch_key.value() == TensorTypeId::QuantizedCPUTensorId) {
+        // If the input is quantized, but the quantization is not supported.
+        TORCH_CHECK(false, "Tried runninig '", operator_name_, "', but it",
```

Comment:
s/runninig/running/

Reply:
You might also want to specify something like "Tried running X *with a quantized tensor* but X expects a non-quantized input" just for clarity